modules/ROOT/pages/aurads/managing-instances/backup-restore-export.adoc

```diff
@@ -1,27 +1,37 @@
 [[aurads-backup-restore-export]]
-= Backup, export and restore
+= Backup, export, and restore
 :description: This page describes how to backup, export and restore your data from a snapshot.
 
-The data in your AuraDS instance can be backed up, exported, and restored using snapshots. A _snapshot_ is a copy of the data in an instance at a specific point in time.
+The data in your AuraDS instance can be backed up, exported, and restored using _snapshots_.
+A snapshot is a copy of the data in an instance at a specific point in time.
 
-The *Snapshots* tab within an AuraDS instance shows a list of available snapshots footnote:[Snapshots are kept in the system for 180 days.]. To access the *Snapshots* tab:
+The *Snapshots* tab within an AuraDS instance shows a list of available snapshots.
+To access the *Snapshots* tab:
 
 . Navigate to the https://console.neo4j.io/?product=aura-ds[Neo4j Aura Console^] in your browser.
 . Select the instance you want to access.
 . Select the *Snapshots* tab.
 
-== Snapshot actions
+== Snapshot types
 
-=== Take a snapshot
+There are two different types of snapshot:
 
-You can manually trigger a snapshot by selecting *Take snapshot* in the *Snapshots* tab. The snapshot status is shown as `In progress` until the snapshot has been created; then, the `Status` becomes `Completed`.
+* *Scheduled* - Runs when you first create an instance, when changes to the underlying system occur (for example, a new patch release), and automatically at a cadence depending on your plan type. 
+Scheduled snapshots are run automatically once a day for Professional instances and once an hour for Enterprise instances.
```

Comment:
```suggestion
* *Scheduled* - Runs when you first create an instance, when changes to the underlying system occur (for example, a new patch release), and automatically once a day.
```

Reply:
Based on us showing only one scheduled snapshot a day for enterprise, I think this is the correct messaging to use.